src/components/VideoPlayer/VideoPlayer.stories.mdx

```diff
@@ -20,15 +26,22 @@ import { getCategory } from "../../utils/componentCategories";
   parameters={{
     design: {
       type: "figma",
-      url:
-        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=10968%3A3680",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=10968%3A3680",
     },
     jest: ["VideoPlayer.test.tsx"],
   }}
   argTypes={{
-    attributes: { table: { disable: true } },
-    className: { table: { disable: true } },
-    modifiers: { table: { disable: true } },
+    aspectRatio: {
+      control: { type: "select" },
+      options: ratiosEnumValues.options,
+    },
+    className: { control: false },
+    id: { control: false },
+    showHelperInvalidText: {
+      table: { defaultValue: { summary: true } },
+    },
+    videoId: { control: false },
```

Comment:
`videoId` and `videoType` don't need to be disabled.  They just need to be null initially.  Someone viewing the `Video` story should be able to update the props.

Reply:
Updated.